src/core/create_app.ts

```diff
@@ -159,9 +163,11 @@ function generate_server_manifest(
 	manifest_data: ManifestData,
 	path_to_routes: string,
 	cwd: string,
+	root: string,
 	src: string,
 	dest: string,
-	dev: boolean
+	dev: boolean,
+	export_build: boolean
```

Comment:
Why don't you just pass in the correct value for `root` (i.e. `cwd` or the `root` directory) instead of adding this variable to check if `root` should be overriden?

Reply:
I thought that it would be more obvious what is happening to the path, than to morph `cwd` to `root` in `build.js` and using it as `root` in `create_app.ts`. If you prefer this approach I could change this. I could also update my sapper version to resolve the merge conflicts.